py_ocpi/core/utils.py

```diff
@@ -18,7 +19,10 @@ def set_pagination_headers(response: Response, link: str, total: int, limit: int
 def get_auth_token(request: Request) -> str:
     headers = request.headers
     headers_token = headers.get('authorization', 'Token Null')
-    return headers_token.split()[1]
+    token = headers_token.split()[1]
+    if token == 'Null':
```

Comment:
you can disable the bandit security check for this line only

Reply:
@HamedAkhavan do you have an example of how to do it? I only was able to find how to skip a whole test together

Reply:
change the line to `if token == 'Null':  # nosec`